sdk/identity/azure-identity/tests/test_default.py

```diff
@@ -93,12 +93,12 @@ def assert_credentials_not_present(chain, *excluded_credential_classes):
         assert actual <= default  # n.b. we know actual is non-empty
         assert default - actual <= excluded
 
-    # with no environment variables set, ManagedIdentityCredential = ImdsCredential
+    # with no environment variables set, ManagedIdentityCredential holds an instance of  ImdsCredential
```

Comment:
Although true, this is irrelevant because the test isn't concerned with the internal state of the credential. Actually this test can now be simplified because the composition of `DefaultAzureCredential` no longer depends on MSI_ENDPOINT. This test need only verify that when `exclude_managed_identity=True`, `ManagedIdentityCredential` isn't present; patching `os.environ` to set/clear MSI_ENDPOINT is unnecessary.